requirements.txt

```diff
@@ -1,3 +1,11 @@
-click
-six
-pyyaml
+click==6.6
+coverage==4.0.3
+funcsigs==1.0.0
+mock==2.0.0
+nose==1.3.7
+ordereddict==1.1
+pbr==1.8.1
+pep8==1.7.0
+pyflakes==1.1.0
+PyYAML==3.11
+six==1.10.0
```

Comment:
@detiber We also have a couple sources of truth for the requirements -- setup.py and the requirements files. I think we need to maintain both; the requirements files for pip and setup.py for anyone installing via setup.py.